Composer/plugins/vacore/template/.gitignore

```diff
@@ -0,0 +1 @@
+/generated/
```

Comment:
Consider putting all the .gitignore paths in the top level one.  It is painful to track down .gitignores at levels lower than the package.json.